Lib/test/test_unittest/test_async_case.py

```diff
@@ -434,6 +434,30 @@ async def cleanup(self, fut):
         test.doCleanups()
         self.assertEqual(events, ['asyncSetUp', 'test', 'cleanup'])
 
+    def test_setup_get_event_loop(self):
+        # See https://github.com/python/cpython/issues/95736
+        events = []
+        class TestCase1(unittest.IsolatedAsyncioTestCase):
+            def setUp(self):
+                self.loop = asyncio.get_event_loop_policy().get_event_loop()
```

Comment:
Why call `asyncio.get_event_loop_policy().get_event_loop()`? Would not `asyncio.get_running_loop()` work as well?

`self.loop` is never used. Why keep this reference?

Reply:
This is part of the issue derived from aiohttp, where I just deleted everything in aiohttp until I had a minimal realistic reproducer

We should still put the GEL-loop somewhere and `self.assertIs(` against the `get_running_loop`